3p/3d-gltf/AnimationLoop.js

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
please use `lower-case-with-dash.js` naming convention for the JS filenames for consistency.

Reply:
done